### PR TITLE
Move Location tabs to buttons, like mobile worker page

### PR DESCRIPTION
**Pull request by esoergel**
@biyeun @benrudolph 

screenshot from 2016-03-28 14-07-20

I think this is a strict improvement over what we've got now, but it could use a couple tweaks that I'll mention inline.

### CODE REVIEW

**Comment by benrudolph** [comment]
:dress: looks good! no tests, mergin

File: corehq/apps/locations/templates/locations/manage/location.html [hunk 1]
```diff
@@ -83,6 +83,25 @@
       </script>
 
       {% if not form.is_new_location %}
+
+        <p class="lead">{{ location.name }} <small>{{ location.location_type_name }}</small></p>
```
Comment (esoergel) [inline corehq/apps/locations/templates/locations/manage/location.html]: Location name and location type could use more visual separation.  @biyeun any thoughts on that?

**Comment by benrudolph** [comment]
oops

File: corehq/apps/locations/templates/locations/manage/location.html [hunk 2]
```diff
@@ -83,6 +83,25 @@
...
+
+        <div class="btn-toolbar" style="margin-bottom: 20px;">
+
+          {% if location.get_id and location.location_type_object.can_have_children %}
+          <a class="btn btn-default" href="{% url "create_location" domain %}?parent={{ location.get_id }}">
```
Comment (esoergel) [inline corehq/apps/locations/templates/locations/manage/location.html]: Think `btn-default` is appropriate for both of these?  They're not the primary action on the page.

File: corehq/apps/locations/templates/locations/manage/location.html (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/locations/templates/locations/manage/location.html]: is this shown in the screen shot? also that space in the HTML might not do anything. should use `&nbsp`

**Comment by esoergel** [comment]
@benrudolph don't worry about it, I can merge a separate change with any feedback from Biyeun.

File: corehq/apps/locations/templates/locations/manage/location.html (hunk 1 above)
Comment (esoergel) [inline corehq/apps/locations/templates/locations/manage/location.html]: Yeah, this is the `Duval County` title on the page.

Comment (benrudolph) [inline corehq/apps/locations/templates/locations/manage/location.html]: ah i see. i agree some sort of separator would be helpful. i read it as Duval County not Duval is of type County

Comment (esoergel) [inline corehq/apps/locations/templates/locations/manage/location.html]: I could throw the location type name in a bootstrap badge or label, or maybe just `<small class="text-muted" />`

Comment (benrudolph) [inline corehq/apps/locations/templates/locations/manage/location.html]: i'm a fan of the bootstrap label. we use that in a lot of places

**Comment by biyeun** [comment]
@esoergel typically we've been using `btn-success` as the class for the button that adds a new item. I'm pro moving to that for the sake of consistency. I think at a later point (soon, I hope) we should establish a better, written set of guidelines for button colors.

Overall, page is vastly improved :+1: Thanks, Ethan! :tada:

**Comment by biyeun** [comment]
@benrudolph speedy gonzales :checkered_flag: :stuck_out_tongue_closed_eyes:

**Comment by benrudolph** [comment]
:runner:

**Pull request merged**